### DISCUSSION TITLE
Proven HVAC Model

### DISCUSSION THREAD

**Discussion by hiknmike007** [esphome-econet/esphome-econet#352] [General]
Thanks for the amazing work on this.  I have a furnace model R96VA0702317MSA (Prestige series) that is working great if you'd like to add it to the proven model list.

**Comment by barndawgie** [comment]
> Thanks for the amazing work on this.  I have a furnace model R96VA0702317MSA (Prestige series) that is working great if you'd like to add it to the proven model list.  

Thanks for the info! Do
You have it in a multi-zone setup? A communicating ODU (Outdoor Unit)?

**Reply by hiknmike007** [reply]
Single zone.  Not sure on the ODU.  I do have an AC unit outside but don't know if it has the communication feature.  I did notice that the outdoor air temp and supply air temp reported to HA were clearly incorrect (6514 degrees F).

**Reply by barndawgie** [reply]
If you check your ESPHome config file you should see a line you can uncomment to try and pull data from the ODU.

Not sure about the temperature issue - you can try opening a GitHub issue with a log file.

**Comment by FoxBluff** [comment]
I just started using ESPHome econet with our new furnace and heat pump.   Wow, this is amazing.  I'm not really sure where to list our equipment,  so I thought I'd add it on this discussion thread.  Thank you for all of the amazing work.  If you want any information from me based on my installation,  please let me know.  More than happy to help.

Furnace:  Sure (Rheem) S98MV0703A17UPSJAP
Heat Pump: RD18AY3620250827417
Zone Controller: REPNL700ZON
Main Tstat: CETST800SYS
2nd Tstat: RECTL800ZON

I am getting information from the ODU including Kwh.  However, my furnace Kwh is at 0 always as are in BTU attributes.   The flame mA does have a reading.  I seem to be getting the rest of the information including zone names, zone CFM numbers,  etc.

Again, thank you so much for everyone's effort in putting this together.  It truely is amazing!

Bill

I can include screen shoot of the HA dashboard,  etc

**Comment by FoxBluff** [comment]
I stand corrected.  I recompiled and redeployed and now only indoor Kw is stuck at 0.  Other than that, it's great.  I'll look at recompiling with the log level set at debug and see if there is anything obvious.
